### ISSUE TITLE
Weird Output on NixOS

### ISSUE THREAD

**Issue by AlexanderMaxRanabel** [iVacon/mrfetch#3]
ı dont think this should be the output:
```
choroalp@nixos

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
┐ OS: EPORT_URL=https://github.com/NixOS/nixpkgs/issues // I think its a good idea to find an another way to get distro name
│ Kernel: 6.1.33
│ Mem: 1/3 GB (2 GB Available)
│ Uptime: 11 minutes
│ Shell: /run/current-system/sw/bin/nu // Looks ugly
┘ CPU: Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz
```

**Comment by iVacon** [comment]
Shell part is solved by PR #2, I'll work on the distro name now.

**Comment by iVacon** [comment]
Issue is 100% fixed by commit 34ab056.
